packages/turf-dissolve/index.js

```diff
@@ -29,118 +24,49 @@ import { closestGreaterNumber } from "./lib/get-closest";
  * //addToMap
  * var addToMap = [features, dissolved]
  */
-function dissolve(featureCollection, options) {
+function dissolve(fc, options) {
   // Optional parameters
   options = options || {};
   if (!isObject(options)) throw new Error("options is invalid");
-  var propertyName = options.propertyName;
+  const propertyName = options.propertyName;
 
   // Input validation
-  collectionOf(featureCollection, "Polygon", "dissolve");
+  collectionOf(fc, "Polygon", "dissolve");
 
   // Main
-  var fc = clone(featureCollection);
-  var features = fc.features;
-
-  var originalIndexOfItemsRemoved = [];
-
-  features.forEach(function (f, i) {
-    f.properties.origIndexPosition = i;
-  });
-  var tree = rbush();
-  tree.load(fc);
-
-  for (var i in features) {
-    var polygon = features[i];
-
-    var featureChanged = false;
-
-    tree.search(polygon).features.forEach(function (potentialMatchingFeature) {
-      polygon = features[i];
-
-      var matchFeaturePosition =
-        potentialMatchingFeature.properties.origIndexPosition;
-
+  const outFeatures = [];
+  if (!options.propertyName) {
+    return featureCollection([
+      multiPolygon(
+        polygonClipping.union(...fc.features.map((f) => f.geometry.coordinates))
```

Comment:
Arrow functions also will probably fail the build

Reply:
The spread operator is also failing the es5 check, but that's the way polygonClipping accepts multiple polygons, need to think there if there a good way around that :(

Reply:
polygonClipping.union.apply or .call I can't remember which offhand. On mobile atm.

Reply:
Have replaced with `.apply()`